docs/3.0rc/deploy/infrastructure-concepts/work-pools.mdx

```diff
@@ -156,106 +184,136 @@ If you don't use the `--type` flag to specify an infrastructure type, you are pr
+                                   (**) denotes a late run
+```
 
-Optional configuration parameters you can specify to update the work pool include:
+### Work pool status
 
-| Option                                             | Description                                                                                                                                                |
-| -------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| <span class="no-wrap">`--base-job-template`</span> | The path to a JSON file containing the base job template to use. If unspecified, Prefect uses the default base job template for the given worker type. |
-| `--description`                                    | A description of the work pool.                                                                                                                            |
-| `--concurrency-limit`                              | The maximum number of flow runs to run simultaneously in the work pool.                                                                                    |
+Work pools have three statuses:
 
-<Tip>
-**Managing work pools in CI/CD**
+- `READY`: A work pool has at least one online worker sending heartbeats to the work pool.
+- `NOT_READY`: A work pool has no online workers.
+- `PAUSED`: You can place a work pool in a paused status manually or through an automation. When a paused work pool is unpaused, it is reassigned the 
+appropriate status based on whether any workers are sending heartbeats.
 
-  Version control your base job template by committing it as a JSON file to your repository. And control updates to your 
-  work pools' base job templates with the `prefect work-pool update` command in your CI/CD pipeline. For example, you 
-  use the following command to update a work pool's base job template to the contents of a file named `base-job-template.json`:
+### Pause and delete work pools
 
-    ```bash
-    prefect work-pool update --base-job-template base-job-template.json my-work-pool
-    ```
-</Tip>
+You can pause a work pool at any time to stop the delivery of work to workers. Workers will not receive any work when polling a paused pool.
 
-#### Base job template
+To pause a work pool through the Prefect CLI, use the `prefect work-pool pause` command:
 
-Each work pool has a base job template that allows the customization of the behavior of the worker executing flow runs from the work pool.
+```bash
+prefect work-pool pause 'test-pool'
+```
 
-The base job template acts as a contract defining the configuration passed to the worker for each flow run and the options available 
-to deployment creators to customize worker behavior per deployment.
+To resume a work pool through the Prefect CLI, use the `prefect work-pool resume` command with the work pool name.
```

Comment:
might be worth a `<Note>` in this section to mention that pausing work pools does _not_ pause underlying deployments which may continue to get scheduled